src/middleware/authorization.js

```diff
@@ -0,0 +1,16 @@
+/* Determines API permission levels based on user role */
+const authorization = (permittedRoles) => (req, res, next) => {
+  const { user } = req;
+
+  /* If the user's role is found in the array of permitted roles,
+   * the user is granted access
+   */
+  if (user && (permittedRoles.includes(user.role) || user.role === 'admin')) {
```

Comment:
I'm okay with this but cam we remove the last part and just include `admin` as a necessary role. I think it's just better to be explicit so we don't forget how much power admin has/needs to have.

Reply:
When you say include `admin` as a necessary role, do you mean to place it in the `permittedRoles` array?

Reply:
as in just removing the last clause `|| user.role === 'admin')`